packages/list/test/mwc-list.test.ts

```diff
@@ -1444,6 +1444,8 @@ suite('mwc-list:', () => {
         await items[1].updateComplete;
         await element.updateComplete;
 
+        await new Promise((res) => setTimeout(res, 50));
```

Comment:
This is going to cause lots of issues migrating users. Is there a way we can deterministically know when items is ready? Perhaps expose an `itemsReady: Promise<ListItem[]>` that gets chained by the debounce? (super skeleton code) e.g.

```ts
debounce(...) => {
  element.itemsReady.then(() => {fn(...);return element.items});
}
class {
  ...
  itemsReady: Promise.resolve([]);
}
```

Reply:
oh also you can override

```ts
async _getUpdateComplete() {
  await super._getUpdateComplete();
  await this.itemsReady;
}
```

so `list.updateComplete` should include `itemsReady`